STEER/STEER/AliGenerator.h

```diff
@@ -89,28 +89,32 @@ class AliGenerator : public TNamed, public AliRndm
 	{ox=fOrigin.At(0);oy=fOrigin.At(1);oz=fOrigin.At(2);}
     virtual void GetOrigin(TLorentzVector &o) const
 	{o[0]=fOrigin.At(0);o[1]=fOrigin.At(1);o[2]=fOrigin.At(2);o[3]=0;}
-  // Stack 
+  // Stack
     void SetStack (AliStack *stack) {fStack = stack;}
     AliStack* GetStack(){return fStack;}
   // Collision Geometry
     virtual Bool_t ProvidesCollisionGeometry() const {return kFALSE;}
-    virtual Bool_t NeedsCollisionGeometry()    const {return kFALSE;}    
+    virtual Bool_t NeedsCollisionGeometry()    const {return kFALSE;}
     virtual AliCollisionGeometry* CollisionGeometry() const {return fCollisionGeometry;}
     virtual void SetCollisionGeometry(AliCollisionGeometry* geom) {fCollisionGeometry = geom;}
 
     virtual Float_t GetEnergyCMS() const { return fEnergyCMS; }
     virtual void    SetEnergyCMS(Float_t energy = 0) { fEnergyCMS = energy; }
     virtual void    GetProjectile(TString& tar, Int_t& a, Int_t& z) const
-    {tar = fProjectile; a = fAProjectile; z = fZProjectile;}    
+    {tar = fProjectile; a = fAProjectile; z = fZProjectile;}
     virtual void    GetTarget(TString& tar, Int_t& a, Int_t& z) const
-    {tar = fTarget; a = fATarget; z = fZTarget;}    
+    {tar = fTarget; a = fATarget; z = fZTarget;}
     virtual void    SetProjectile(TString proj="", Int_t a = 0, Int_t z = 0)
 	{fProjectile = proj; fAProjectile = a; fZProjectile = z;}
     virtual void    SetTarget(TString tar="", Int_t a = 0, Int_t z = 0)
 	{fTarget = tar; fATarget = a; fZTarget = z;}
-    
+
     virtual void    SetSeed(UInt_t seed);
-    
+
+    ///-------------------------------------------------------------------------
+    virtual void SetUserTrigger(Bool_t (*fn)(AliStack*)) { fSetUserTrig = kTRUE; fUserTrigger = fn; }     // Enable user trigger
```

Comment:
Hello @lucamicheletti93 , @shahor02 ,

Shouldn't variables starting with "f" be private members of the class in AliRoot? Or it is only that all private members start with "f", but not all variables that start with "f" are private members?

Chiara

Reply:
In aliroot the data members start with fCapitalized, but there is not policy on function argument names.

Reply:
Ok, thanks for confirming!